screens/Settings/AboutScreen/AboutScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { Text, View, Image } from "react-native"
+import { styles } from "./styles"
+import { globalStyles } from "../../../assets/global/globalStyles"
+import { useSafeAreaInsets } from "react-native-safe-area-context"
+import { BackArrow } from "../../../components/BackArrow/BackArrow"
+
+export const AboutScreen = () => {
+  const insets = useSafeAreaInsets()
+  const developers = [
+    "Aidas Venckunas",
+    "Alberto Centonze",
+    "Gäel Conde",
+    "Alexandre Mourot",
+    "Gustave Charles",
+    "Gaspard Thoral",
+    "Pedro Laginhas",
+  ]
+
+  return (
+    <View style={[styles.container, { paddingTop: insets.top }]}>
+      <Text style={globalStyles.boldText}>About us</Text>
```

Comment:
The about us text layout is not good enough. There should be more free space around and it should be bigger

Reply:
How can the text be bigger and still have more free space around it?